.github/workflows/pr_presubmit.yaml

```diff
@@ -23,20 +23,34 @@ jobs:
     runs-on: ${{matrix.os}}
     steps:
     - uses: actions/checkout@v2
+
     - uses: actions/setup-go@v3
       with:
         go-version: 1.x
         cache: true
+
     - id: files
       uses: Ana06/get-changed-files@v2.2.0
-    - uses: actions/checkout@v2
+
+    - name: Get labels
```

Comment:
[nit] can we have this be right above the run tests step that actually uses these labels